+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ grep expect
+ rpm -qa
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Tue Nov 21 13:19:19 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL criu
+ pkgs=criu
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs criu --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 0:16:34 ago on Tue 21 Nov 2023 01:03:14 PM CST.
No match for argument: criu
Error: Unable to find a match: criu'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 0:16:34 ago on Tue 21 Nov 2023 01:03:14 PM CST.
No match for argument: criu
Error: Unable to find a match: criu'
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Tue Nov 21 13:19:50 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Nov 21 13:19:50 2023 - INFO  - Start to run test.
+ criu check -vNUM
oe_test_criu_base_check_02.sh: line 30: criu: command not found
+ CHECK_RESULT 127 0 0 'Check criu check -vNUM failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check criu check -vNUM failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check criu check -vNUM failed'
+ LOG_ERROR 'Check criu check -vNUM failed'
+ message='Check criu check -vNUM failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check criu check -vNUM failed'
Tue Nov 21 13:19:51 2023 - ERROR - Check criu check -vNUM failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_criu_base_check_02.sh line 31'
+ message='oe_test_criu_base_check_02.sh line 31'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_criu_base_check_02.sh line 31'
Tue Nov 21 13:19:51 2023 - ERROR - oe_test_criu_base_check_02.sh line 31
+ '[' 0 -eq 1 ']'
+ return 0
+ criu check --verbosity=NUM
oe_test_criu_base_check_02.sh: line 32: criu: command not found
+ CHECK_RESULT 127 0 0 'Check criu check --verbosity=NUM failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check criu check --verbosity=NUM failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check criu check --verbosity=NUM failed'
+ LOG_ERROR 'Check criu check --verbosity=NUM failed'
+ message='Check criu check --verbosity=NUM failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check criu check --verbosity=NUM failed'
Tue Nov 21 13:19:52 2023 - ERROR - Check criu check --verbosity=NUM failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_criu_base_check_02.sh line 33'
+ message='oe_test_criu_base_check_02.sh line 33'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_criu_base_check_02.sh line 33'
Tue Nov 21 13:19:52 2023 - ERROR - oe_test_criu_base_check_02.sh line 33
+ '[' 0 -eq 1 ']'
+ return 0
+ criu check --display-stats
oe_test_criu_base_check_02.sh: line 34: criu: command not found
+ CHECK_RESULT 127 0 0 'Check criu check --display-stats failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check criu check --display-stats failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check criu check --display-stats failed'
+ LOG_ERROR 'Check criu check --display-stats failed'
+ message='Check criu check --display-stats failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check criu check --display-stats failed'
Tue Nov 21 13:19:53 2023 - ERROR - Check criu check --display-stats failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_criu_base_check_02.sh line 35'
+ message='oe_test_criu_base_check_02.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_criu_base_check_02.sh line 35'
Tue Nov 21 13:19:54 2023 - ERROR - oe_test_criu_base_check_02.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Nov 21 13:19:54 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Nov 21 13:19:55 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue Nov 21 13:19:56 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 0:16:34 ago on Tue 21 Nov 2023 01:03:14 PM CST.
No match for argument: criu
Error: Unable to find a match: criu ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 0:16:34 ago on Tue 21 Nov 2023 01:03:14 PM CST.
No match for argument: criu
Error: Unable to find a match: criu'
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 192, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 163, in rpm_remove
    with open(tmpfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Last metadata expiration check: 0:16:34 ago on Tue 21 Nov 2023 01:03:14 PM CST.\nNo match for argument: criu\nError: Unable to find a match: criu'
+ '[' 0 -ne 0 ']'
+ rm -rf FILE
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue Nov 21 13:19:59 2023 - INFO  - End to restore the test environment.
